cmake/std/atdm/apps/sparc/SPARCTrilinosPackagesList.cmake

```diff
@@ -26,4 +25,5 @@ SET(SPARC_Trilinos_Packages
   SEACAS
   Gtest
   ROL
+  TriKota
```

Comment:
@e10harvey, since we are not cloning the Dakota source, we should not be enabling TriKoka because it can't build.